### PR TITLE
Maintanence/2224 npm audit

### PR DESCRIPTION
**Pull request by RAnderson96**
Upgrades Cypress version and addresses npm audit vulnerabilities.

Issue: Scottish-Natural-Heritage/Licensing#2224

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,28 +22,28 @@
   },
   "license": "MIT",
   "dependencies": {
+    "axios": "^1.7.2",
     "express": "^4.18.1",
-    "morgan": "^1.10.0",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.2",
```
Comment (paulcoxsnh) [inline package.json:27]: `jsonwebtoken` version 9.0.0 and above introduces a bug, we need to leave it at 8.5.1

**Review by paulcoxsnh — CHANGES_REQUESTED** [review]
Need to downgrade `jsonwebtoken`.

File: package.json (hunk 1 above)
Comment (RAnderson96) [reply package.json]: Has now been reverted, good spot

**Review by paulcoxsnh — APPROVED** [review]
All good :)

**Pull request merged**